data_utilities/text_align.py

```diff
@@ -104,3 +106,50 @@ def initialize_indices(word_list, segments):
         if not retry:
             break
     return indices
+
+
+class compare_breaks(object):
+
+    def __init__(self, a, b, markers=['α', 'β']):
+        # a, b: lists of strings (sections
+        # markers: pair of chars that aren't contained in the strings
+        assert all(x not in y for x in markers for y in a+b), 'markers should not be in text'
+        assert len(markers) == 2 and markers[0] != markers[1], 'markers should be two different charts'
+        self.a = a
+        self.b = b
+        self._marker_a = markers[0]
+        self._marker_b = markers[1]
+        self.insert_break_marks()
+
+    def insert_break_marks(self):
```

Comment:
this method doesn't return anything, although the documentation says it does. It instead adds members to the class - members that are not defined in the `__init__` method. If this is a setup method, mark it as protected by renaming to `_insert_break_marks`. Make sure to set `self.a_with_b_breaks` to a placeholder value (`None` is fine) in the `__init__`. 

The data set by this method should be marked as protected by adding an underscore as well. I'd then recommend making a _getter_ method that returns the relevant data